src/hotspot/share/jfr/instrumentation/jfrEventClassTransformer.cpp

```diff
@@ -1151,14 +1151,14 @@ static ClassFileStream* schema_extend_event_klass_bytes(const InstanceKlass* ik,
   initialize_symbols();
   static const u2 public_final_flag_mask = JVM_ACC_PUBLIC | JVM_ACC_FINAL;
   const ClassFileStream* const orig_stream = parser.clone_stream();
-  assert(orig_stream != NULL, "invariant");
+  assert(orig_stream != nullptr, "invariant");
   const int orig_stream_length = orig_stream->length();
   // allocate an identically sized buffer
-  u1* const new_buffer = NEW_RESOURCE_ARRAY_IN_THREAD_RETURN_NULL(THREAD, u1, orig_stream_length);
-  if (new_buffer == NULL) {
-    return NULL;
+  u1* const new_buffer = NEW_RESOURCE_ARRAY_IN_THREAD_RETURN_nullptr(THREAD, u1, orig_stream_length);
```

Comment:
fix

Reply:
Propagating the term "nullptr" is weird when it is used conceptually, like for example in comments and macros.